src/main/resources/db/changelog/changes/v1.0.4/sql/derived-tables/create-view-purchase-order-lines.sql

```diff
@@ -0,0 +1,56 @@
+CREATE OR REPLACE VIEW drv_purchase_order_line_details
+AS
+SELECT po.id,
+    po.jsonb ->> 'poLineNumber'::text AS purchase_order_line_number,
+    po.jsonb ->> 'receiptStatus'::text AS purchase_order_line_receipt_status,
+    po.jsonb ->> 'paymentStatus'::text AS purchase_order_line_payment_status,
+    (po.jsonb -> 'metadata'::text) ->> 'createdDate'::text AS purchase_order_line_created_date,
+    (po.jsonb -> 'metadata'::text) ->> 'updatedDate'::text AS purchase_order_line_update_date,
+    (po.jsonb -> 'metadata'::text) ->> 'createdByUserId'::text AS purchase_order_line_creater_id,
```

Comment:
Can we change purchase_order_line_creater_id ->  purchase_order_line_created_by_id? This would keep it consistent with purchase_order_created_by_id below.